main.py

```diff
@@ -37,30 +37,7 @@
 
 dataset_tracker: Optional[dset.DatasetTracker] = None
 
-class ScheduleInfo(pydantic.BaseModel):
-    """Scheduled queue information.
-    
-    Fields:
-        updated_time: The time when the current schedule was updated, in the format of time.time().
-        queue: A dictionary with queued experiments.
-          Each key is a RID, and its value is the dictionary with the experiment information:
-          "due_date", "expid", "flush", "pipeline", "priority", "repo_msg", and "status".
-    """
-    updated_time: Optional[float] = None
-    queue: Optional[dict[int, dict[str, Any]]] = None
-
-    def update(self, queue: dict[int, dict[str, Any]]):
-        """Updates the schedule info.
-        
-        Args:
-            queue: A new scheduled queue.
-        """
-        self.updated_time = time.time()
-        self.queue = queue
-
-
-latest_schedule = ScheduleInfo()
-
+schedule_tracker: Optional[schd.ScheduleTracker] = None
```

Comment:
How about writing these global assignments without empty lines between each line, just like the imports?